src/loose/index.js

```diff
@@ -41,9 +41,7 @@ defaultOptions.tabSize = 4
 
 // eslint-disable-next-line camelcase
 export function parse_dammit(input, options) {
-  let p = new LooseParser(input, options)
-  p.next()
-  return p.parseTopLevel()
```

Comment:
I think it'd be beneficial if their APIs aligned a bit more. It seems kind of odd for them to be different in little ways here and there.